definitions/json/CaseField/CaseField.json

```diff
@@ -1287,7 +1287,7 @@
   },
   {
     "CaseTypeID": "ET_EnglandWales",
-    "ID": "etInitialConsiderationBeforeYouStart",
+    "ID": "initialConsiderationBeforeYouStart",
```

Comment:
this also needs to be on CaseEventToFields - https://github.com/hmcts/et-ccd-definitions-englandwales/blob/ffc1472ef35a327427337d557b833770b0f42495/definitions/json/CaseEventToFields/CaseEventToFields.json#L3538C38-L3538C38

Reply:
Done